services/utils/persistence/src/test/java/org/eclipse/ditto/services/utils/persistence/mongo/MongoClientWrapperTest.java

```diff
@@ -54,6 +52,27 @@ private static String createUri(final boolean sslEnabled) {
         return connectionString.getConnectionString();
     }
 
+    @Test
+    public void createByUriWithExtraSettings() {
```

Comment:
This test is not related to the changes made to the `MongoReadJournal`, right?
We can of course keep it .. just wanted to make sure why you added the test :)

Reply:
Yes, its a long story but in the beginning I was not sure if the config by URI worked correctly. So, yes, the test is not related but I think it makes sense to keep.